### ISSUE TITLE
/bin/tar: unrecognized option '--transform=s/.*\///'

### ISSUE THREAD

**Issue by stepanovdg** [pocopico/tinycore-redpill#18]
Trying to run './rploader.sh build broadwell-7.0.1-42218 static'

`[#] Verifying /home/tc/redpill-load/custom/extensions/thethorgroup.jg.virtio/ds3617xs_42218/virtio-4.4-108.tgz file... [OK]
[#] Unpacking files from /home/tc/redpill-load/custom/extensions/thethorgroup.jg.virtio/ds3617xs_42218/virtio-4.4-108.tgz to /home/tc/redpill-load/custom/extensions/thethorgroup.jg.virtio/ds3617xs_42218/... [ERR]
[-] Failed to unpack tar

/bin/tar: unrecognized option '--transform=s/.*\///'
BusyBox v1.33.0 (2021-01-17 14:57:11 UTC) multi-call binary.

Usage: tar c|x|t [-ZzJjahmvokO] [-f TARFILE] [-C DIR] [-T FILE] [-X FILE] [OPTION]... [FILE]...

Create, extract, or list files from a tar file

	c	Create
	x	Extract
	t	List
	-f FILE	Name of TARFILE ('-' for stdin/out)
	-C DIR	Change to DIR before operation
	-v	Verbose
	-O	Extract to stdout
	-m	Don't restore mtime
	-o	Don't restore user:group
	-k	Don't replace existing files
	-Z	(De)compress using compress
	-z	(De)compress using gzip
	-J	(De)compress using xz
	-j	(De)compress using bzip2
	--lzma	(De)compress using lzma
	-a	(De)compress based on extension
	-h	Follow symlinks
	-T FILE	File with names to include
	-X FILE	File with glob patterns to exclude
	--exclude PATTERN	Glob pattern to exclude
	--overwrite		Replace existing files
	--strip-components NUM	NUM of leading components to strip
	--no-recursion		Don't descend in directories
	--numeric-owner		Use numeric user:group
	--no-same-permissions	Don't restore access permissions
[-] Recipe file for extension thethorgroup.jg.virtio platform ds3617xs_42218 is logically invalid - file download https://github.com/jimmyGALLAND/redpill-ext/raw/master/virtio/releases/virtio-4.4.108-broadwell.tgz cannot be unpacked. You SHOULD contact the extension packer to report that (see below)
`

and after 

`[#] Unpacking /home/tc/redpill-load/build/1643766319/pat-ds3617xs_42218-unpacked/rd.gz file to /home/tc/redpill-load/build/1643766319/rd-ds3617xs_42218-unpacked... [ERR]
[!] Failed to unpack compressed ramdisk

*** Process will exit ***
`

Should tar version be updated ?

**Comment by stepanovdg** [comment]
Maybe I am missing something. 
As I can see all linux tools in tinycore are busybox 1.33 and they have limited number of arguments with comparison to usual linux tools.
Tried to build apollolake and broadwell. One is using redpill-load from jumkey another jimmyGALLAND develop
Both repos use syntax not supported by busybox for (tar, gzip, xz, cpio, losetup)
First I tried to change functions in repill-load/include/*.sh (file and boot-load) to use old syntax analogs available in busybox.
It worked for tar, gzip, patch but not worked for xz | cpio. So ramdisk uncomressed was always empty.
I cloned redpill-load and redpill-lvm to fedora 35 box. And was able to build first redpill.ko and proper image.
First tried to mount it using rploader.sh by commenting build part and use my copied image with name loader.img. 
But script still failed with losetup and lspci.

So after I copied it in the same pve image folder and changed vm.conf to use build image as boot. 
Was able to start and find web server with install page. After several attempts install proper pat. But server after reboot always show repair window.
